### ISSUE TITLE
how to update diaspora with this?

### ISSUE THREAD

**Issue by COLABORATI** [eraclitux/ansible-diaspora#7]
looks like there is no way to handle updates with this?

**Comment by eraclitux** [comment]
That's correct, for now I don't have time to implement this. Feel free to to apply PRs with any useful changes.
